docs/changes/table/11660.other.rst

```diff
@@ -0,0 +1,2 @@
+For mixins, ``Table`` shows the object's class' qualitative name, not "object",
+except in the edge case of a blank name.
```

Comment:
```suggestion
Printing a ``Table`` now shows the qualified class name of mixin columns in the
dtype header row  instead of "object". This applies to all the ``Table`` formatted output
methods whenever ``show_dtype=True`` is selected.
```
I think the "edge case" is too contrived to deserve mention here, despite the logical possibility in the code. I am not aware of a reason anyone would make a mixin column class that has a blank class name.